### PR TITLE
HI Centrality DQM bug fix

### PR DESCRIPTION
**Pull request by BetterWang**
Same as #10266 .
This is a bug fix to protect agains missing EventPlane object.

### CODE REVIEW

File: DQM/Physics/BuildFile.xml [hunk 1]
```diff
@@ -21,4 +21,5 @@
 <use   name="RecoJets/JetProducers"/>
 <use   name="RecoJets/JetAlgorithms"/>
 <use   name="DQM/PhysicsHWW"/>
+<use   name="RecoHI/HiEvtPlaneAlgos">
```
Comment (deguio) [inline DQM/Physics/BuildFile.xml]: is this needed?

Comment (BetterWang) [inline DQM/Physics/BuildFile.xml]: Probable not. I can remove it.

Comment (deguio) [inline DQM/Physics/BuildFile.xml]: ok, thanks,
F.

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**